src/db/commentToDB.php

```diff
@@ -3,43 +3,72 @@
 session_start();
 	//Author: Jasen Ratnam
 	
-	
-	//get current user id
-	if (isset($_SESSION["userID"])) {
-		$loggenOnUser = $_SESSION["userID"];
-		echo "Found User: ", $loggenOnUser, "<br />";
-	} else {
-		$loggenOnUser = " a public user";
+	global $p_id;
+	//get u_id from session.
+	function fetch_user() {
```

Comment:
Please save REST requests into variables, and then pass them as arguments to the function.